src/PlayFabBuddy.Tests/PlayFabBuddy.Lib.Tests/Aggregate/PlayerAccountAggregateTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using PlayFabBuddy.Lib.Entities.Accounts;
+using Xunit;
+
+namespace PlayFabBuddy.Lib.Aggregate.Tests;
+public class PlayerAccountAggregateTests
+{
+    [Fact()]
+    public void PlayerAccountAggregateTest()
+    {
+        var mainAccount = new MasterPlayerAccountEntity
+        {
+            Id = Guid.NewGuid().ToString()
+        };
+
+        var aggregate = new PlayerAccountAggregate(mainAccount);
+
+        Assert.Equal(mainAccount, aggregate.MasterPlayerAccount);
+
+        var titlePlayerAccount = new TitlePlayerAccountEntity
+        {
+            Id = Guid.NewGuid().ToString()
+        };
+
+        mainAccount.PlayerAccounts = new List<TitlePlayerAccountEntity>() { titlePlayerAccount };
+
+        var newAggregate = new PlayerAccountAggregate(mainAccount);
+
+        Assert.NotNull(newAggregate.MasterPlayerAccount.PlayerAccounts);
+        Assert.Equal(mainAccount, newAggregate.MasterPlayerAccount);
+        Assert.Contains(titlePlayerAccount, newAggregate.MasterPlayerAccount.PlayerAccounts);
+    }
+
+    [Fact()]
+    public void PlayerAccountAggregateTestByIds()
+    {
+        var mainAccountGuid = Guid.NewGuid().ToString();
+        var titleAccountGuid = Guid.NewGuid().ToString();
+
+        var aggregate = new PlayerAccountAggregate(mainAccountGuid, titleAccountGuid);
+
+        Assert.NotNull(aggregate.MasterPlayerAccount);
```

Comment:
These are two tests

Reply:
This test is done do make get rid of compiler warning of possible null, so i would keep it in here :)